docs/recipes/install/common/pxse_enabled_builds.tex

```diff
@@ -46,8 +46,13 @@
 % ohpc_command if [[ ${enable_opa} -eq 1 ]];then
 % ohpc_indent 10
 \begin{lstlisting}[language=bash,keywords={},upquote=true,keepspaces]
+<<<<<<< .merge_file_CHumzE
+# Optionally, choose Omni-Path enabled MPI builds. Otherwise, skip to retain default MPI stacks
+[sms](*\#*) (*\install*) openmpi-psm2-intel-ohpc mvapich2-psm2-intel-ohpc
+=======
 # Optionally, choose the Omni-Path enabled build for MVAPICH2. Otherwise, skip to retain IB variant
 [sms](*\#*) (*\install*) mvapich2-psm2-intel-ohpc
+>>>>>>> .merge_file_juf65L
```

Comment:
there is merge conflict marker in this file

Reply:
Sorry, @adrianreber . I must not have had the latest changes in my local branch when I created the PR. This had nothing to do with what I was working on. I will need to rebase next time.